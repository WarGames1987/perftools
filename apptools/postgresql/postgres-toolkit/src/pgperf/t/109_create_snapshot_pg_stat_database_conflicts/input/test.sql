SELECT pgperf.create_snapshot_pg_stat_database_conflicts(0);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;

SELECT pgperf.create_snapshot_pg_stat_database_conflicts(1);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;

SELECT pgperf.create_snapshot_pg_stat_database_conflicts(2);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;

SELECT pgperf.delete_snapshot_pg_stat_database_conflicts(1);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;

SELECT pgperf.delete_snapshot_pg_stat_database_conflicts(2);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;

SELECT pgperf.delete_snapshot_pg_stat_database_conflicts(0);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;
