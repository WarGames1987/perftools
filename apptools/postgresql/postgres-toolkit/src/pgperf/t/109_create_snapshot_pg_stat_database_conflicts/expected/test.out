SELECT pgperf.create_snapshot_pg_stat_database_conflicts(0);
 create_snapshot_pg_stat_database_conflicts 
--------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;
 count 
-------
     1
(1 row)

SELECT pgperf.create_snapshot_pg_stat_database_conflicts(1);
 create_snapshot_pg_stat_database_conflicts 
--------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;
 count 
-------
     2
(1 row)

SELECT pgperf.create_snapshot_pg_stat_database_conflicts(2);
 create_snapshot_pg_stat_database_conflicts 
--------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;
 count 
-------
     3
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_database_conflicts(1);
 delete_snapshot_pg_stat_database_conflicts 
--------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;
 count 
-------
     2
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_database_conflicts(2);
 delete_snapshot_pg_stat_database_conflicts 
--------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;
 count 
-------
     1
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_database_conflicts(0);
 delete_snapshot_pg_stat_database_conflicts 
--------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database_conflicts;
 count 
-------
     0
(1 row)

